src/main/java/io/leangen/graphql/GraphQLSchemaGenerator.java

```diff
@@ -1090,7 +1090,7 @@ public TypeResolver getTypeResolver(GraphQLUnionType unionType) {
 
         @Override
         public DataFetcher<?> getDataFetcher(GraphQLFieldsContainer parentType, GraphQLFieldDefinition fieldDef) {
-            return codeRegistry.getDataFetcher(parentType, fieldDef);
+            return codeRegistry.getDataFetcher(FieldCoordinates.coordinates(parentType, fieldDef), fieldDef);
```

Comment:
I don't see any difference in `GraphQLCodeRegistry#getDataFetcher` between graphql-java 21 and 22. Why did you have to change these?